src/bugjira/issue.py

```diff
@@ -0,0 +1,22 @@
+from typing import Any
+
+from pydantic import BaseModel
+
+
+class Issue(BaseModel):
+    """BaseModel representing either a bugzilla bug or a jira issue.
+    Future refactoring will allow access to the bug/issue's attributes
+    via instances of this class.
+    """
+
+    key: str
+    bugzilla: Any
```

Comment:
I still feel these are not needed. A user can easily see what kind of issue this is by using either isinstance(), or type() if they don't know the options. However, again, I will not hold the patch over a difference of design approach.